app/grants/views.py

```diff
@@ -1473,13 +1473,23 @@ def grant_edit(request, grant_id):
         if logo:
             grant.logo = logo
 
-        twitter_handle_1 = request.POST.get('handle1', None)
+        twitter_handle_1 = request.POST.get('handle1', '')
+        twitter_handle_2 = request.POST.get('handle2', '')
```

Comment:
^ why not default it to None?

Reply:
from models, ;-)

```
twitter_handle_1 = models.CharField(default='', max_length=255, help_text=_('Grants twitter handle'), blank=True)
```

default is '' either way... no `null=True`